_includes/breadcrumbs.html

```diff
@@ -0,0 +1,14 @@
+{% if page.namespace != 'index' %}
+  <div id="breadcrumbs">
+    {% assign crumbs = page.url | remove: '/index.html' | split: '/' %}
```

Comment:
can remove `remove: '/index.html' |` since the url never includes that in our cases?